WORKSPACE.bazel

```diff
@@ -12,6 +12,14 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("//:third_party_repositories.bzl", "grpc", "protobuf")
 
+# rules_python import has to be higher in the WORKSPACE file because other rules below load an older
```

Comment:
do you know which repository rule in particular?

Reply:
The comment is no longer relevant. Sorry, I forgot to delete it. I will do it immediately.